fastlane/Fastfile

```diff
@@ -570,15 +592,23 @@ REPOSITORY_NAME="WordPress-Android"
     }
 
     delete_old_changelogs(build: options[:build_number])
-    download_path=Dir.pwd + "/metadata/android"
-    gp_downloadmetadata(project_url: "https://translate.wordpress.org/projects/apps/android/release-notes/",
+    download_path = File.join(Dir.pwd, paths[:metadata_dir], 'android')
+    gp_downloadmetadata(
+      project_url: paths[:gp_url],
       target_files: files,
-      locales: RELEASE_NOTES_LOCALES,
+      locales: RELEASE_NOTES_LOCALES, # Currently the same for WP and JP
       source_locale: "en-US",
-      download_path: download_path)
+      download_path: download_path
+    )
 
+    if app == 'jetpack'
+      # For WordPress, the en-US release notes come from what's downloaeded from GlotPress' en-gb locale.
```

Comment:
Anyone knows the history behind this? Why do we have the source release notes (provided by Editorial in English) used as source when uploading to WP's GlotPress… but then have an `en-gb` locale for them in GlotPress, which ends up being mapped to `en-US` during download of all locales (see `RELEASE_NOTES_LOCALES` constant and line 14 of this `Fastfile`), and thus that `en-gb` translation from GlotPress ends up being used as the new source when generating the `release_notes.xml` in that lane?

Why not get rid of `en-gb` for from WP's GlotPress Release Notes project and just use the source copy for `en-US`?!